commands/room.py

```diff
@@ -1,4 +1,7 @@
-"""this is where all room commands goes"""
+"""this is where all room commands goes
```

Comment:
Fix Capitalization please (and start the info line with the file name, like the other files)